config.yaml

```diff
@@ -21,6 +21,7 @@ default:
    --plugin-option=tmpfs:required_ram_mb=39800"
  commit_updates: True
  push_updates: True
+ build_dependencies: True
```

Comment:
My main point here is automating the act of inferring if *each* package must be rebuilt instead of having a global manual option to do it, the commit ID was just the first idea that came in my mind. Better ideas are welcome. =)   Another (better) way is to keep the date of the last modification in the package directory and rebuild if there were any modifications since then. This would cover adding of files to SOURCES, changes in the spec and changes in the yaml. The only case this approach would not cover is a modification to the builds git repo, but then you could explicitly use the option to clean the cache I've mentioned before.